packages/modules/gcloud/src/pubsub-emulator-container.ts

```diff
@@ -0,0 +1,44 @@
+import { AbstractStartedContainer, GenericContainer, Wait } from "testcontainers";
+import type { StartedTestContainer } from "testcontainers";
+
+const EMULATOR_PORT = 8085;
+const CMD = "gcloud beta emulators pubsub start --host-port 0.0.0.0:8085";
+const DEFAULT_IMAGE = "gcr.io/google.com/cloudsdktool/google-cloud-cli";
+
+export class PubSubEmulatorContainer extends GenericContainer {
+  private _projectId?: string;
+
+  constructor(image = DEFAULT_IMAGE) {
+    super(image);
+
+    const commandLine = `${CMD} --project=${this._projectId ?? "test-project"}`;
+    this.withExposedPorts(EMULATOR_PORT)
+      .withWaitStrategy(Wait.forLogMessage(/Server started/g, 1))
+      .withCommand(["/bin/sh", "-c", commandLine])
+      .withStartupTimeout(120_000);
+  }
+
+  public withProjectId(projectId: string): PubSubEmulatorContainer {
+    this._projectId = projectId;
```

Comment:
Similar to issue with the other module, the project ID doesn't have any effect as the `command` has already been set.

Reply:
Looks like I need to move it to the `start`-function then. I will see if that will work

Reply:
@cristianrgreco Updated the code and added the requested test